### DISCUSSION TITLE
Weekly report 19.10 - 26.10 2023

### DISCUSSION THREAD

**Discussion by Umbranecis** [SE-TINF22B6/Plapy#53] [Weekly reports]
Goals for this Week:

**Commands:**

1. Implement new Command "Radio" to play Random music
2. Create new Commands for Playlist administration

**Documentation**

1. Create a readable setup Documentation on how to run the bot locally
2. Create a process to easily track progress and create weekly reports

**Database**

1. Setup a basic Database for persistence
4. Create a docker image to run the local database
5. Write first values from Command execution in database

**Comment by derrrobin** [comment]
I like the way you document your weekly updates. For me its clearly what you have done and what are you next steps. Great job :)

**Comment by nuo-li** [comment]
Please also report progress of your functional requirement analysis.
Reminder of your tasks in this week: UML use case diagrams, explanation of the use cases (via related user stories), UI mockup, and start writing SRS.
